sklearn/naive_bayes.py

```diff
@@ -512,8 +512,40 @@ class _BaseDiscreteNB(_BaseNB):
 
     __init__
     _joint_log_likelihood(X) as per _BaseNB
+    _update_feature_log_prob(alpha)
+    _count(X, Y)
     """
 
+    @abstractmethod
+    def _count(self, X, Y):
+        """Update counts that are used to calculate probabilities.
+
+        The counts make up a sufficient statistic extracted from the data.
+        Accordingly, this method is called each time ``fit`` or ``partial_fit``
```

Comment:
Nit:

```suggestion
        Accordingly, this method is called each time `fit` or `partial_fit`
```

Trying to move to single backticks in general. If the docstring was rendered into HTML, double and single backticks do the same thing.